cleanlab/internal/constants.py

```diff
@@ -21,3 +21,25 @@
     2 * FLOATING_POINT_COMPARISON
 )  # lower bound imposed to clip confident thresholds from below, has to be larger than floating point comparison
 TINY_VALUE = 1e-100  # very tiny value for clipping
+
+
+# Object Detection Constants
+EUC_FACTOR = 0.1  # factor to control magnitude of euclidian distance
+MAX_ALLOWED_BOX_PRUNE = 0.97  # This is max allowed percent of prune for boxes below threshold before a warning is thrown.
+
+ALPHA = 0.9  # param for objectlab, weight between IoU and distance when considering similarity matrix. High alpha means considering IoU more strongly over distance
+LOW_PROBABILITY_THRESHOLD = 0.001  # param for objectlab, lowest prediction threshold allowed when considering predicted boxes to identify badly located label boxes
+HIGH_PROBABILITY_THRESHOLD = 0.5  # param for objectlab, high probability threshold for considering predicted boxes to identify overlooked and swapped label boxes
+TEMPERATURE = 0.1  # param for objectlab, temperature of the softmin function where a lower score suggests softmin acts closer to min
+
+OVERLOOKED_THRESHOLD = (
+    0.3  # threshold to determine max score for a box to be considered an overlooked issue
```

Comment:
clarify what kind of threshold these are:

eg "per-box label quality score threshold" or something like that